product_docs/docs/pgd/5/admin-manual/installing/07-configure-proxies.mdx

```diff
@@ -19,7 +19,7 @@ It is best practice to configure PGD Proxy for clusters to enable this behavior.
 To set up a proxy, you will need to first prepare the cluster and sub-group the proxies will be working with by:
 
 * Logging in and setting the `enable_raft` and `enable_proxy_routing` node group options to `true` for the sub-group. Use [`bdr.alter_node_group_option`](/pgd/latest/reference/nodes-management-interfaces#bdralter_node_group_option), passing the sub-group name, option name and new value as parameters.
-* Create as many uniquely named proxies as you plan to deploy using [`bdr.create_proxy`](/pgd/latest/reference/routing#bdrcreate_proxy) and passing the new proxy name and the sub-group it should be attached to.
+* Create as many uniquely named proxies as you plan to deploy using [`bdr.create_proxy`](/pgd/latest/reference/routing#bdrcreate_proxy) and passing the new proxy name and the sub-group it should be attached to. The `bdr.create_proxy` sets default values for different proxy options like listen_port which defaults to 6432.
```

Comment:
Not sure how this addition helps the reader. 

```suggestion
* Create as many uniquely named proxies as you plan to deploy using [`bdr.create_proxy`](/pgd/latest/reference/routing#bdrcreate_proxy) and passing the new proxy name and the sub-group it should be attached to. The `bdr.create_proxy` does not create a proxy, but creates a space for a proxy to register itself with the cluster. The space contains configuration values which can be modified later. Initially it is configured with default proxy options such as setting the listen_port to 6432.
```

Reply:
> but creates a space for a proxy to register itself with the cluster

Not sure I get this. It is same as create node or create group (which creates nothing but group or node configs). It creates default configuration. But may be you meant same. Committing it.